examples/routingmanagerd/CMakeLists.txt

```diff
@@ -2,22 +2,31 @@
 # terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain
 # one at http://mozilla.org/MPL/2.0/.
 
+cmake_minimum_required(VERSION 3.15)
+
+
+option(VSOMEIP_INSTALL_ROUTINGMANAGERD "Whether or not to install the routing manager daemon.")
+
 # Daemon
-add_executable(routingmanagerd routingmanagerd.cpp)
+add_executable(routingmanagerd)
+target_sources(routingmanagerd PRIVATE routingmanagerd.cpp)
+
+if(TARGET Genivi::dlt)
+  target_compile_definitions(routingmanagerd PRIVATE USE_DLT)
+  target_link_libraries(routingmanagerd PRIVATE Genivi::dlt)
+endif()
```

Comment:
why not keep the ${DLT_LIBRARIES}?

this seems to increase complexity

Reply:
I thought I removed this change.  This is a bad change that should never have been pushed.

Reply:
Fixed in 7cc8a1a7